test/fixtures/class-comment-annotation/actual.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ParentComponent from './Parent'
+
+export default class Greeting extends ParentComponent {
```

Comment:
What about simplifying the **tests**? For instance, I like how we can better reduce noise with:
```js
class Foo1 extends ParentComponent {
  render() {}
}

Foo1.propTypes /* remove-proptypes */ = {
  bar1: PropTypes.string,
};
```